### ISSUE TITLE
Slicing Stops @ ~10%

### ISSUE THREAD

**Issue by Karithay** [Ultimaker/Cura#14448]
### Application Version

5.2.2, 5.2.1, 5.0

### Platform

Windows 11 / AMD Ryzen 7 5800 8-Core Processor 3.40 GHz / 16.0 GB RAM / AMD RX 6800 XT16GB GDDR6

### Printer

Creality CR-6 SE & Mingda Magician X

### Reproduction steps

Loaded the file AEPWAR01 - 4x4 Chinampas Half (Empty).stl for slicing.

Hit the Slice button

### Actual results

The slicing progress gets to ~10%
![Cura-Chinampa Bug 01](https://user-images.githubusercontent.com/124118230/215945908-f50b7d30-74da-43e3-9344-c2c9a7abdce9.jpg)

Then slicing stops and it throws up an error box, asking me to click the link to report the bug
![Cura-Chinampa Bug 02](https://user-images.githubusercontent.com/124118230/215945893-5285578c-9a7a-4801-b9f3-1418fb591cd2.jpg)

This is not the only file in this set that this error occurred with, but some of the files, even those with plants,  did slice (and print) normally)

This was attempted in 5.2.2, 5.2.1, 5.0, and 4.1 for both my Creality CR6-SE and my Mingda Magician X. It only actually sliced the file in Cura 4.1

### Expected results

It should have sliced normally so that I could print the file.

### Checklist of files to include

- [X] Log file
- [X] Project file

### Additional information & file uploads

5.2 Log [cura5.2.log](https://github.com/Ultimaker/Cura/files/10553345/cura5.2.log)
5.0 Log [cura5.0.log](https://github.com/Ultimaker/Cura/files/10553346/cura5.0.log)
4.1 Log [cura.log](https://github.com/Ultimaker/Cura/files/10553348/cura.log)
Project file [AEPWAR01 - 4x4 Chinampas Half (Empty).zip](https://github.com/Ultimaker/Cura/files/10553354/AEPWAR01.-.4x4.Chinampas.Half.Empty.zip)

This bug is similar to that in https://github.com/Ultimaker/Cura/issues/14441, as it mentions ending at ~10%

**Comment by GregValiant** [comment]
Thanks for the report and the files.
When I zoom in on the model in Cura I get this:
![image](https://user-images.githubusercontent.com/64202104/216059040-758147cf-6652-445e-934b-6e1af5a10fdf.png)

So although the model is "watertight" there appears to be something going on internally.  The repair utilities I use didn't really note any errors though.  The model should have sliced.

There is an often reported bug in Cura 5.x that struggles with models like this that have "dirt" or "tile" floors.  There are a LOT of non-planar surfaces all very close to being on the same plane.  A fix for this was added into Cura a few days ago to address the bug and 5.3 will be available shortly with the fix.  Hopefully it works.

A workaround for the bug was to move and/or rotate the model on the Cura build plate.  This wouldn't slice for me when I opened the project, but did after I moved the model a ways towards the left front.
![image](https://user-images.githubusercontent.com/64202104/216060694-af9cf31d-a8cf-4ff1-8045-dbb46fa1c660.png)

I'll mark this as a duplicate "slicing error" and close it as the Cura team has a LOT of evidence of what goes on there and it has been addressed.  I hope you understand.
